react/ContactsListModal/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react'
+import { Query, fetchPolicies } from 'cozy-client'
+import ContactsList from '../ContactsList'
+import Modal, { ModalHeader, ModalDescription, ModalBackButton } from '../Modal'
+import Spinner from '../Spinner'
+import styles from './styles.styl'
+import Input from '../Input'
+import PropTypes from 'prop-types'
+import withBreakpoints from '../helpers/withBreakpoints'
+
+const thirtySeconds = 30000
+const olderThan30s = fetchPolicies.olderThan(thirtySeconds)
+
+const mkFilter = filter => contacts => {
```

Comment:
Nit: filter could be renamed filterStr ? I thought filter was a function at first.